kratos/containers/sparse_graph.h

```diff
@@ -26,6 +26,7 @@
 // Project includes
 #include "includes/define.h"
 #include "utilities/parallel_utilities.h"
+#include <span/span.hpp>
```

Comment:
If external should be in external includes, otherwise replace <> with ""

Reply:
no, by using < > you are telling to consider it as a system include and not to show the eventual warnings

Reply:
In that case should be in System includes, not in Project includes

Reply:
ok